lib/internal/socket_list.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
```

Comment:
I'd like it if we could move the exports up here, a la:

```javascript
module.exports = {SocketListSend, SocketListReceive}
```

*(We have shorthand object literals now, so we can take advantage of that!)*